src/controller.rs

```diff
@@ -151,11 +151,13 @@ impl AppState {
             .await
             .expect("ok");
 
-        let default_subaccount = wallet.sub_account(default_subaccount_id.unwrap_or(0));
-        if let Err(err) = client.subscribe_account(&default_subaccount).await {
-            log::error!(target: LOG_TARGET, "couldn't subscribe to user updates: {err:?}");
-        } else {
-            log::info!(target: LOG_TARGET, "subscribed to subaccount: {default_subaccount}");
+        for sub_account_id in sub_account_ids.clone() {
```

Comment:
its better to iter the borrow and just copy the value later if needed e.g. `*sub_account_id`
```suggestion
        for sub_account_id in sub_account_ids {
```